osmoutils/sumtree/tree.pb.go

```diff
@@ -167,26 +167,26 @@ func init() {
 func init() { proto.RegisterFile("osmosis/store/v1beta1/tree.proto", fileDescriptor_72b0b7af579d13be) }
 
 var fileDescriptor_72b0b7af579d13be = []byte{
-	// 297 bytes of a gzipped FileDescriptorProto
```

Comment:
why are these changing? (it's happening in a few protos)

Reply:
probably because our proto gen docker file has been changed since this proto file was last generated . ( I needed to regenerate proto files due to our import paths for osmomath, osmoutils changing)